sympy/polys/domains/tests/test_domains.py

```diff
@@ -570,6 +580,17 @@ def test_Domain_get_exact():
     assert QQ.frac_field(x, y).get_exact() == QQ.frac_field(x, y)
 
 
+def test_Domain_characteristic():
+    for F, c in [(FF(3), 3), (FF(5), 5), (FF(7), 7)]:
+        for R in F, F[x], F.frac_field(x), F.old_poly_ring(x), F.old_frac_field(x):
```

Comment:
R is not used in this loop interior -- is something missing?

Reply:
Ah, good point. Fixed.